templates/vue/src/components/Tapestry.vue

```diff
@@ -1,17 +1,42 @@
 <template>
   <div id="tapestry">
-    <div class="d-flex justify-content-center mb-3" style="padding: 30vh 0;" v-if="!tapestryLoaded">
+    <div
+      v-if="!tapestryLoaded"
+      class="d-flex justify-content-center mb-3"
+      style="padding: 30vh 0;"
+    >
       <label>Loading Tapestry </label>
-      <b-spinner type="grow" variant="secondary" small style="margin: 5px 5px 5px 20px;"></b-spinner>
-      <b-spinner type="grow" variant="primary" small style="margin: 5px;"></b-spinner>
-      <b-spinner type="grow" variant="danger" small style="margin: 5px;"></b-spinner>
+      <b-spinner
+        type="grow"
+        variant="secondary"
+        small
+        style="margin: 5px 5px 5px 20px;"
+      ></b-spinner>
+      <b-spinner
+        type="grow"
+        variant="primary"
+        small
+        style="margin: 5px;"
+      ></b-spinner>
+      <b-spinner
+        type="grow"
+        variant="danger"
+        small
+        style="margin: 5px;"
+      ></b-spinner>
     </div>
-    <SettingsModal :tapestryApiClient="TapestryAPI" @settings-updated="handleSettingsUpdate" />
-    <RootNodeButton v-show="tapestryLoaded && !tapestry.rootId" @add-root-node="addRootNode" />
+    <SettingsModal
+      :tapestry-api-client="TapestryAPI"
+      @settings-updated="handleSettingsUpdate"
+    />
+    <RootNodeButton
+      v-show="tapestryLoaded && !tapestry.rootId"
+      @add-root-node="addRootNode"
+    />
     <NodeModal
       :node="populatedNode"
-      :modalType="modalType"
-      :rootNodeTitle="selectedNode.title"
+      :modal-type="modalType"
+      :root-node-title="selectedNode.title"
```

Comment:
This doesn't break anything?

Reply:
I didn't know this either, but apparently not.

See the [vue docs here](https://vuejs.org/v2/guide/components-props.html#Prop-Casing-camelCase-vs-kebab-case).